packages/shared-ux/page/kibana_template/impl/src/page_template_inner.tsx

```diff
@@ -25,45 +25,60 @@ const getClasses = (template?: string, className?: string) => {
  * A thin wrapper around EuiPageTemplate with a few Kibana specific additions
  */
 export const KibanaPageTemplateInner: FC<Props> = ({
-  template,
   className,
   pageHeader,
   children,
   isEmptyState,
+  pageSideBar,
+  pageSideBarProps,
   ...rest
 }) => {
-  /**
-   * An easy way to create the right content for empty pages
-   */
-  const emptyStateDefaultTemplate = 'centeredBody';
-  let header = pageHeader;
+  let header;
 
-  if (isEmptyState) {
-    if (pageHeader && !children) {
-      template = template ?? emptyStateDefaultTemplate;
-      const { iconType, pageTitle, description, rightSideItems } = pageHeader;
-      const title = pageTitle ? <h1>{pageTitle}</h1> : undefined;
-      const body = description ? <p>{description}</p> : undefined;
-      header = undefined;
-      children = (
-        <EuiEmptyPrompt
-          iconType={iconType}
-          iconColor="" // This is likely a solution or app logo, so keep it multi-color
-          title={title}
-          body={body}
-          actions={rightSideItems}
-        />
-      );
-    } else if (pageHeader && children) {
-      template = template ?? 'centeredContent';
-    } else if (!pageHeader) {
-      template = template ?? emptyStateDefaultTemplate;
-    }
+  if (isEmptyState && pageHeader && !children) {
+    const { iconType, pageTitle, description, rightSideItems } = pageHeader;
+    const title = pageTitle ? <h1>{pageTitle}</h1> : undefined;
+    const body = description ? <p>{description}</p> : undefined;
+    children = (
+      <EuiPageTemplate.EmptyPrompt
+        iconType={iconType}
+        iconColor="" // This is likely a solution or app logo, so keep it multi-color
+        title={title}
+        body={body}
+        actions={rightSideItems}
+      />
+    );
+  } else if (pageHeader) {
+    header = <EuiPageTemplate.Header {...pageHeader} />;
   }
 
-  const classes = getClasses(template, className);
+  let sideBar;
+  if (pageSideBar) {
+    sideBar = (
+      <EuiPageTemplate.Sidebar
+        // TODO: Get `offset` from Kibana Chrome Header and pass directly to:
```

Comment:
@constancecchen Should we log an issue for this TODO?  I'm thinking so, and then reference it here.